### ISSUE TITLE
Not accessible with screen readers like JAWS and NVDA

### ISSUE THREAD

**Issue by pranavkaushik** [CodeSeven/toastr#500]
When the toastr pops out the screen reader couldn't catch the toastr message. This can be resolved by adding aria-live="assertive" and role="alert" for the toastr message tag.

Kindly update the same.

Thanks.

**Comment by blimey85** [comment]
This would be a good thing to support. @pranavkaushik, can you add this in and then create a pull request so it can be merged. Thanks!

**Comment by elmoaaron** [comment]
Very important!

**Comment by Nindouja** [comment]
I opened the pull request https://github.com/CodeSeven/toastr/pull/598 to fix that.

**Comment by AdrienNguyenWorldline** [comment]
+1 on that behaviour, there is no drawback in adding the right aria tags
PR should be merged to the main project
